files.js/modal.js

```diff
@@ -0,0 +1,165 @@
+      <div class="popupcontlangs">
+        <ul>
+          <li>${projectList[projectList.length - 1].html}</li>
+          <li>${projectList[projectList.length - 1].bootstrap}</li>
+          <li>${projectList[projectList.length - 1].ruby}</li>
+        </ul>
+      </div>
+      <div class="popupcontbody">
+        <div class="popupcontbodyleft">
+          <img src="${projectList[projectList.length - 1].projectImage}" alt="image">
+        </div>
+        <div class="popupcontbodyright">
+          <p class="desktophide">${projectList[projectList.length - 1].projectDesc}</p>
+          <p class="mobilehide">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.</p>
+          <div class="popupcontbodyrightbtns">
+          <a href="https://carlylechia.github.io/MY-PORTFOLIO/"><button type="button" id="modalbtns live">See Live <img src="images/viewlive.png" alt="view live"></button></a>
+          <a href="https://github.com/carlylechia/MY-PORTFOLIO.git"><button type="button" id="modalbtns live">See Source <img src="images/viewsource.png" alt="view Source"></button></a>
+          </div>
+        </div>
+      </div>
+    </div>
+  `;
+  popUp.innerHTML = showing;
+  body.appendChild(popUp);
+
+  document.addEventListener('click', (e) => {
+    if (e.target.id === 'closeModal') {
+      popUp.style.display = 'none';
+    }
+  });
+}
+
+document.addEventListener('click', (e) => {
+  if (e.target.id === 'proj1') {
+    modalpopup();
+  }
+});
+featuredbtn.onclick = modalpopup;
+
+const projectsSection = document.querySelector('.my-projects');
+function loopdiv() {
+  projectsSection.innerHTML = '';
+  for (let i = 0; i < projectList.length - 1; i += 1) {
+    projectsSection.innerHTML = `${projectsSection.innerHTML}
+  <div class="project">
+           <div class="projectHeader">
+             <h3>${projectList[i].projectHeader}</h3>
+           </div>
+           <div class="projectBody">
+             <p>
+             ${projectList[i].projectDesc}
+             </p>
+             <ul>
+               <li class="projectHTML">
+                 <ul>
+                  <li>${projectList[i].projectHtml}</li>
+                 </ul>
+               </li>
+               <li class="projectBootstrap">
+                 <ul>
+                   <li>${projectList[i].projectbootstrap}</li>
+                 </ul>
+               </li>
+               <li class="projectRuby">
+                 <ul>
+                   <li>${projectList[i].projectRuby}</li>
+                 </ul>
+               </li>
+             </ul>
```

Comment:
- [OPTIONAL]
 The same here I recommend you to use technologies (HTML, CSS ...) as an array in your `projectList` data, and then you can use `for loop` to iterate over them and keep your logic drier (DRY Principe - Don't Repeat Yourself).